composer.json

```diff
@@ -18,13 +18,13 @@
     ],
 
     "require": {
-        "php": ">=5.3.1",
-        "behat/mink":  "~1.7",
-        "symfony/phpunit-bridge": "^4.0"
```

Comment:
this one should be kept as a requirement. We want it to be installed when using the testsuite, not only when developing the testsuite itself.

Reply:
moved bridge back to required.

Reply:
reverted and updated phpunit-bridge, waiting for results https://travis-ci.org/minkphp/MinkSelenium2Driver/builds/549177247?utm_source=github_status&utm_medium=notification